### PR TITLE
Add commented-out HTTPS configuration section

### PR DESCRIPTION
**Pull request by jwbernin**
In recent versions of Firefox and (especially) Chrome, ".dev" domains are
force-redirected to a secure connection (HTTPS) instead of HTTP. Not
supporting an HTTPS connection makes for a bad user experience, so we want
to make enabling SSL support easy. Since we don't issue certificates, this
section is commented out by default, and it will require modification.

This is done primarily for PiAware, though it is applicable to just the
Dump1090 project.

This functionality has been tested with a LetsEncrypt certificate on PiAware 3.8.1.

### CODE REVIEW

File: debian/lighttpd/89-dump1090-fa.conf [hunk 1]
```diff
@@ -25,3 +25,32 @@ server.modules += ( "mod_setenv" )
...
+
+# Uncomment this section to enable SSL traffic (HTTPS) - especially useful
+# for .dev domains
+## Listen on 8443 for SSL connections
+#server.modules += ( "mod_openssl" )
+#$HTTP["host"] == "piaware.example.com" {
+#  $SERVER["socket"] == ":8443" {
+#    ssl.engine = "enable"
+#    ssl.pemfile = "/root/ssl_certificate/combined.pem"
+#    ssl.ca-file =  "/root/ssl_certificate/fullchain.cer"
```
Comment (mutability) [inline debian/lighttpd/89-dump1090-fa.conf:37]: Maybe put these in `/etc/lighttpd/`

**Comment by jwbernin** [comment]
Those locations are just examples. The certs shouldn't go in /etc/lighttpd; the most common location is in /etc/ssl/certs and /etc/ssl/private in Debian (on which Raspberry OS is based). That location will probably change if the end user uses LetsEncrypt certificates, so in many ways I would rather leave the /root/ssl_certificates path in there, and make sure the documentation surrounding PiAware mentions changing this path.

**Comment by mutability** [comment]
Sure, I appreciate that they're just examples, but you wouldn't want to put them in `/root` on a real install so it's not a great default even for an example. `/etc/ssl/` would be fine if you think that matches better.

**Comment by jwbernin** [comment]
I've updated my branch and the PR with a modified example path.

File: debian/lighttpd/89-dump1090-fa.conf (hunk 1 above)
Comment (jwbernin) [reply debian/lighttpd/89-dump1090-fa.conf]: Changed to `/etc/ssl/certs`

**Comment by mutability** [comment]
Sorry for the long delay in merging this

**Pull request merged**